### PR TITLE
test: Cleanup repositories tests (HMS-4658)

### PR DESCRIPTION
**Pull request by regexowl**
Move and clean up Custom repositories tests, split into several commits for better readability.

~~There are still 'On Recreate' tests that will need to be re-enabled, but this will require changes to the code as well (disabling 'Create blueprint' button for blueprints with invalid repos etc.) so leaving that for a separate PR.~~ will be moved to a separate PR

### CODE REVIEW

File: src/test/Components/CreateImageWizard/steps/Repositories/Repositories.test.tsx [hunk 1]
```diff
@@ -225,45 +360,126 @@ describe('Repositories edit mode', () => {
...
+    user.click(removeAnywayBtn);
+
+    await waitFor(() =>
+      expect(screen.queryByText(/Are you sure?/)).not.toBeInTheDocument()
+    );
+
+    await waitFor(() => expect(repoCheckbox).not.toBeChecked());
+  });
 });
+
```
Comment (mgold1234) [inline src/test/Components/CreateImageWizard/steps/Repositories/Repositories.test.tsx:397]: maybe for now we can move all the comment rows? if thats a cleanup pr :)

Comment (mgold1234) [reply src/test/Components/CreateImageWizard/steps/Repositories/Repositories.test.tsx:397]: I mean for now, if you move it to new file, IMOO it is better to move it without comments?

Comment (regexowl) [reply src/test/Components/CreateImageWizard/steps/Repositories/Repositories.test.tsx]: I've removed the "On Recreate" tests.

**Review by mgold1234 — APPROVED** [review]
/lgtm

**Pull request merged**